docs/secrets.md

```diff
@@ -24,3 +24,18 @@ The ID form can be used when there are multiple matching keys.
 example:
 `kops delete secret sshpublickey admin`
 
+### adding secret from file
+
+Secret spec
+```
+apiVersion: kops/v1alpha2
+kind: Secret
+metadata:
+  labels:
+    kops.k8s.io/cluster: dev.k8s.example.com
+spec:
+  name: "admin"
```

Comment:
metadata already has a name field, so I think we should be using that name field.